solutions/lamp-stack-vm-v1/azuredeploy.json

```diff
@@ -11,7 +11,7 @@
             "metadata": {
                 "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
             },
-            "defaultValue": "https://raw.githubusercontent.com/Azure/solution-center/main/solutions/lamp-stack-vm-v1/"
+            "defaultValue": "https://raw.githubusercontent.com/Azure/LAMP/master/"
```

Comment:
This file isn't required and will be deleted, sine we now refer to it directly form the Azure/Lamp repo